charts/greptimedb-cluster/values.yaml

```diff
@@ -488,3 +560,52 @@ debugPod:
     limits:
       memory: 256Mi
       cpu: 200m
+
+# -- The monitoring bootstrap configuration
+monitoring:
+  # -- Enable monitoring
+  enabled: false
+
+  # -- Configure the standalone instance for storing monitoring data
+  standalone: {}
+
+  # -- Configure the logs collection
+  logsCollection:
+    # -- The greptimedb pipeline for logs collection
+    pipeline:
+      data: ""
+
+  # -- Configure vector for logs and metrics collection.
+  vector:
+    # -- vector image registry
+    registry: docker.io
+    # -- vector image repository
+    repository: timberio/vector
+    # -- vector image tag
+    tag: nightly-alpine
+
+    # -- vector resource
+    resource:
```

Comment:
By default, vector does not set resource, can be show as:
```
resource{}
 # requests:
 #   cpu: "50m"
 #   memory: "64Mi"
 # limits:
 #   cpu: "50m"
 #   memory: "64Mi"
```

Reply:
We use `vector` as sidecar and should limit the resource by default.